prisma/schema.prisma

```diff
@@ -273,3 +275,25 @@ model UserInRoom {
 
   @@id([userId, roomId])
 }
+
+model Notification {
+  id Int @id @default(autoincrement())
+
+  auction   Auction @relation(fields: [auctionId], references: [id])

```

Comment:
I guess insted of `auction` and `user` we need add json object with data notification. More details in discord.